app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -265,4 +263,10 @@ Learn more about:
 
 If you have general questions about the information on this page, you can email us at teach.inengland@education.gov.uk.
 
-For help with individual applications to training providers or schools, please [register with a teacher training adviser](/teacher-training-adviser/sign_up/identity). Advisers can give you personalised support with your application.
+You may be eligible for personalised support with your application to a training provider or school from a [teacher training adviser](/teacher-training-advisers).
+
+To check if you're eligible for an adviser, please call us on +44 800 389 2500. Calls will be charged at your country’s standard rate. Or you can contact us using the live chat service at the bottom of this page.
```

Comment:
@laurahermionetennant92 same as previous comment, you can call us so it's consistent with the other page?